src/pages/en/community/nsfw-policy.mdx

```diff
@@ -0,0 +1,206 @@
+- **`NSFL` includes:** Animal abuse, gore, vore, suicide, serious injuries, body horror, etc
+
+Additionally, SFW ("Safe For Work") and SFL ("Safe For Life") can be used as opposing designations for content that definitely doesn't come under the designations of NSFW or NSFL respectively.
+
+NSFL content is (and always will be) roundly banned at Quilt, aside from users seeking support from each other regarding how they feel about it, or for things that have happened to them. With NSFW content, though, the devil's in the details -- this term is loose by design, and they way it's applied differs greatly depending on the people, situation, and content in question.
+
+## Quilt's Definitions
+
+In order to make this easier to understand, we've split potentially NSFW content into several categories:
+
+- **Explicitly Allowed**: Content which we do not consider NSFW, and that we allow in any space
+- **Contextual**: Content which we may allow or disallow based on the discussion it's part of and context around it
+- **Restricted**: Content which is only allowed within the spaces we specifically designate for it
+- **Banned**: Content which is not allowed in all circumstances
+
+Some of our spaces may also have specific rules that apply to them. Where that's the case, we've listed them under the appropriate heading below.
+
+**Note:** It's always worth remembering that everyone has their own experiences and history. Additionally, note that _posting_ content and _discussing_ it isn't the same thing -- but if you're in doubt, you can always contact us via **@ModMail**.
+
+**Additionally:** We are not the ultimate arbiters of what should and should not be considered NSFW content on a global scale. Some types of content we've listed below are included because a user had questions about it, because we want to ensure people understand how posting specific content impacts their safety, or because we wanted to be clear and avoid confusing users -- not because we consider it problematic or we think other people shouldn't be allowed to peruse it, aside from a few specific examples in the banned category.
+
+<Foldable id="space-specific rules">
+<span slot="title">Space-Specific Rules</span>
+
+The following rules apply to specific Quilt community spaces. If a space isn't listed here, you can assume that it follows the rest of this document. **Banned content remains banned regardless of space-specific rules.**
+
+- **Forums:** Minimal contextual content outside designated spaces
+- **GitHub:** No contextual or restricted content
+- **Toolchain Discord:** No restricted content and minimal contextual content
+</Foldable>
+
+<Foldable id="explicitly-allowed-content">
+<span slot="title">Explicitly Allowed Content</span>
+
+The following content **is allowed** in our spaces.
+
+### Anywhere
+
+- Hormone replacement therapy, related discussions, and information on how to obtain it (DIY or otherwise, regardless of local legal context)
```

Comment:
> ...regardless of local legal context)

I'm sure this was discussed a lot internally ect., but I am obligated to formally object here. I understand morality != legality, and that the world may be unfair and full of injustices, however Quilt spaces can include some platforms that also prohibit encouraging illegal activity (namely Discord).

https://discord.com/guidelines
![image](https://user-images.githubusercontent.com/68503800/200784202-492c4472-8050-412b-aee5-3ec1b3aa492f.png)
https://discord.com/terms#8
![image](https://user-images.githubusercontent.com/68503800/200785273-52d4f7e1-0957-4654-be83-240f3d8f2601.png)

Such a policy encourages community members and Community Team members to risk getting their account suspended or terminated, along with potential legal issues in their respective country (messaging platforms can be used as evidence against someone).

Reply:
I know it's tempting to just say "screw it, we'll just see if this causes any issues", but the potential consequences here are rather severe. Encouraging people to break local laws should not be taken lightly (especially if those people do not fully understand what those consequences may be, such as hefty fines they cannot afford)

Reply:
While I appreciate your concerns, I think they can largely be addressed by being responsible with the information we provide. There's no reason for anyone to be deliberately not told that testosterone is a controlled substance in some countries, and it's impossible to talk about DIY while excluding transmasc people -- and that's a good thing.

Quilt doesn't exist to avoid activism, it's one of the reasons we exist.

Reply:
I've had a chat on the DMD and been advised on how to approach this, so I've updated the document accordingly.